rust/rpc_messages/src/dtos/wallets/key_create.rs

```diff
@@ -0,0 +1,9 @@
+use rsnano_core::{Account, PublicKey, RawKey};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct KeyCreateDto {
```

Comment:
The original name was KeyPairDto, which is better, because we use this to return keypairs. The name KeyCreateDto implies that this is a command, but it used as return type instead